commons/commons-model-extended/src/main/kotlin/ebi/ac/uk/extended/model/ExtSubmissionExtensions.kt

```diff
@@ -11,3 +13,6 @@ val ExtSubmission.allFiles
 
 val ExtSubmission.allFileListSections
     get(): List<ExtFileList> = allSections.mapNotNull { it.fileList }
+
+val ExtSubmission.releaseDate
+    get(): String = releaseTime?.toStringDate() ?: ""
```

Comment:
is this actually making us lost information, now we do not know when the submission was released and when not. Nullable type is perfect for this so we should continue using it.